src/voice-pe/customizing/grove-port.md

```diff
@@ -6,7 +6,9 @@ zendesk:
   labels: voice pe, customizing
 ---
 
-Connecting a sensor to the Grove port of the device allows the sensor to use the device's power supply and Wi-Fi connection. Home Assistant can then detect the sensor and start using it. This page provides configuration details for a few selected sensors.
+The [Grove port](https://wiki.seeedstudio.com/Grove_System/) provides an easy way to connect a sensor; it facilitates both a power and data connection. With a minor modification to the VPE's configuration, Home Assistant can then detect the sensor and start using it. This page provides configuration details for a few selected sensors.
```

Comment:
Home Assistant? Wouldn't that be ESPHome?

Reply:
Im not sure, most (if not all) other ESPHome devices can be used without Home Assistant.
If that is not the case with this device it does not matter.
And it is unrelated to this PR, just something I noticed.

Reply:
thanks for the catch. I'll consider it and cover that in another PR.